www/shims/jquery.html.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+/**
+ * @param {FetcherIndex} index to use for URL resolution
+ * @param {Object} context e.g. jQuery, $.fn, etc
+ * @param {String} method name e.g. 'html'
+ * @returns {Function} the shim
+ */
+
+module.exports = function (index, context, method) {
+  var oldMethod = context[method];
+
+  var fixSrc = function (whole, value) {
+    var fixValue;
+    if (value && typeof value === 'string') {
+      fixValue = index.resolveLocalUrl(value);
+      whole = whole.replace(value, fixValue);
+    }
+    return whole;
+  };
+
+  var fixHtml = function (value) {
+    if (value && typeof value === 'string') {
+      value = value.replace(/<img[-_\s\=\"\'\.\/\w]* src="([-_\.\/\w]+)"/gmi, fixSrc);
+    }
```

Comment:
Just checking, but using https://regex101.com and the console, the regex it doesn't match `<img src="http://www.work.com/hello/bye.jpg" />` but it will match `<img src="//www.work.com/hello/bye.jpg" />`

I'm not sure if having the `http(s):` matters for this project (I've got no idea what the input is) ???